### PR TITLE
Updated Art Reg ticket Json structure & code review

### PR DESCRIPTION
**Pull request by VitaliyVK777**
Updated to latest master changes
all tickets tests are successful except of mn_governance.py

### CODE REVIEW

File: src/mnode/mnode-rpc.cpp [hunk 1]
```diff
@@ -1953,6 +1963,56 @@ As json rpc
...
+
+          SecureString strKeyPass;
+          strKeyPass.reserve(100);
+          strKeyPass = params[5].get_str().c_str();
+
+          CArtRoyaltyTicket artRoyaltyTicket =
+            CArtRoyaltyTicket::Create(artTnxId, newPastelID, pastelID, strKeyPass);
+          std::string txid = CPastelTicketProcessor::SendTicket(artRoyaltyTicket);
+
+          mnObj.pushKV(RPC_KEY_TXID, txid);
```
Comment (akobrin1) [inline src/mnode/mnode-rpc.cpp:2014]: move(txid)

**Review by akobrin1 — CHANGES_REQUESTED** [review]

File: src/mnode/mnode-rpc.cpp (hunk 1 above)
Comment (VitaliyVK777) [reply src/mnode/mnode-rpc.cpp:2014]: ok

**Review by akobrin1 — DISMISSED** [review]
Looks good

File: qa/rpc-tests/mn_tickets.py [hunk 2]
```diff
@@ -482,68 +489,80 @@ def personal_pastelid_ticket_tests(self, skip_low_coins_tests):
...
         coins_after = self.nodes[self.non_mn6].getbalance()
         print(f"id ticket price - {self.id_ticket_price}")
         assert_equal(coins_after, coins_before - self.id_ticket_price)  # no fee yet
 
+        print("Personal non_mn6 royalty initialize tested")
+
+    # ===============================================================================================================
+    def personal_nonmn7_green_initialize_tests(self):
+        # personal green PastelID ticket
+        self.nonmn7_green_pastelid1 = self.nodes[self.non_mn7].pastelid("newkey", "passphrase")["pastelid"]
```
[the remark this answers is not in the document]
Comment (VitaliyVK777) [reply qa/rpc-tests/mn_tickets.py:542]: ok. good

**Review by a-ok123 — APPROVED** [review]

**Review by vadymeng0 — APPROVED** [review]

**Review by a-ok123 — APPROVED** [review]

**Pull request merged**